<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_CreateProjet</class>
 <widget class="QWidget" name="w_CreateProjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creaza proiect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Creaza un proiect nou</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="lb_Vat_Mode">
        <property name="toolTip">
         <string>Mod TVA</string>
        </property>
        <property name="text">
         <string>Mod TVA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_Name">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_caen_code">
        <property name="text">
         <string>Cod CAEN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_CIF">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_NRC">
        <property name="toolTip">
         <string>Numar inregistrare la Registrul Comertului</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>NRC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_CIF">
        <property name="toolTip">
         <string>Cod de identificare fiscala</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>CIF</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pb_Cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Creeaza</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_NRC">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lb_namel">
        <property name="toolTip">
         <string>Numele societatii</string>
        </property>
        <property name="text">
         <string>Nume</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_CAEN">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pb_Create">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Inapoi</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cb_VAT_Mode">
        <property name="toolTip">
         <string>Mod TVA</string>
        </property>
        <item>
         <property name="text">
          <string>TVA La Facturare</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TVA La incasare</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Neplatitor TVA</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
